.circleci/config.yml

```diff
@@ -31,6 +31,9 @@ jobs:
       - run:
           name: Show installed Python packages
           command: pip list
+      - run:
+          name: Type checking
+          command: mypy
```

Comment:
Is it possible to follow the style of the other `run` steps below, by explicitly setting `working_directory` and also running `mypy` with the explicit args of `project/wikipron project/tests project/data`?